Update README.md

fahadadeel opened the pull request:
## Contributor License Agreement

Thank you for your contribution! 🚀  
Please ensure you've signed the Contributor License Agreement (CLA) before this PR can be merged.

You can sign the CLA [here](https://cla-assistant.io/fileformat-cells/FileFormat.Cells-for-.NET).

fahadadeel committed:
Update README.md

fahadadeel committed:
Merge branch 'fileformat-cells:main' into main

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/fileformat-cells/FileFormat.Cells-for-.NET?pullRequest=24) <br/>All committers have signed the CLA.

fahadadeel committed:
feat: Add SetHyperlink method to Cell class for hyperlink support

- Implemented `SetHyperlink` method to allow adding hyperlinks to cells with optional tooltips.
- Integrated OpenXML `Hyperlink` and `Hyperlinks` elements to manage cell hyperlinks.
- Added `GetWorksheetPart` helper to locate the parent `WorksheetPart` for a cell.
- Included XML documentation for `SetHyperlink` and `GetWorksheetPart` methods for API clarity.
- Ensured compatibility with the existing `Cell` class and OpenXML SDK conventions.

Example Usage:
- Add a hyperlink to a cell:
  cell.PutValue("Click Here");
  cell.SetHyperlink("https://example.com", "Visit Example Website");
